src/libraries/System.Data.OleDb/src/OleDbDataReader.cs

```diff
@@ -2273,7 +2274,6 @@ internal void DumpToSchemaTable(UnsafeNativeMethods.IRowset? rowset)
             using (OleDbDataReader dataReader = new OleDbDataReader(_connection, _command, int.MinValue, 0))
             {
                 dataReader.InitializeIRowset(rowset, ChapterHandle.DB_NULL_HCHAPTER, IntPtr.Zero);
-                // TODO-NULLABLE: BuildSchemaTableInfo asserts that rowset isn't null, but doesn't do anything with it
```

Comment:
didn't seem actionable, perhaps we could remove the asserts in the BuildSchemaTableInfo but I'll leave it for the owner as I'm not sure why they were added in the first place